templates/index.html

```diff
@@ -1,37 +1,50 @@
+<!DOCTYPE html>
 <html>
 <head>
     <link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='styles/style.css') }}">
-    <link href='https://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-<script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext" rel="stylesheet" type="text/css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.21/lodash.min.js"></script>
+    <script>
+        function refreshPage()
+        {
+            window.location.reload(true);
+        }
+    </script>
+    <title>Tainted Cain Explorer</title>
 </head>
 <body>
 <div class="grid" id="grid1">
-    <aside class="page-rightbar" id="rb1">
-        <div><button onclick="flush_ui()">Search</button></div>
+    <div style="position: fixed; top: 0; right: 0; padding: 5px; width: 200px; background-color: #e44d4d;">
+        <div>
+            <button id="pause">Pause</button>
+            <button id="resume">Resume</button>
+        </div>
         <div id="morebutton"></div>
         <div id="send"></div>
         <div id="stats"></div>
-        A lot of code is taken from <a href="https://github.com/frto027/IsaacDisassemblyRate">this repository</a>
-    </aside>
+        A lot of code is taken from <a href="https://github.com/frto027/IsaacDisassemblyRate" target="_blank">this repository</a>
+    </div>
     <div class="page-main">
         <h1>Tainted Cain Recipes</h1>
-        <input id="search" class="searchbar"></input>
+        <p class="note">
+            <b>NOTE!</b><br>
+            The number of recipes is limited to 10 per item for efficiency reasons.<br>
+            Remember that each item has a lot more recipes. If you want to find another <span onclick="refreshPage()" class="clickable">refresh page</span>
+        </p>
```

Comment:
this is probably overwritten by some other party of code, I don't even see it after page reload

Reply:
I added this in one commit and deleted in another